Connectors/Tekla/Speckle.Connector.Tekla2024/Bindings/TeklaSelectionBinding.cs

```diff
@@ -0,0 +1,70 @@
+using Speckle.Connectors.DUI.Bindings;
+using Speckle.Connectors.DUI.Bridge;
+using Tekla.Structures.Model;
+
+namespace Speckle.Connector.Tekla2024.Bindings;
+
+public class TeklaSelectionBinding : ISelectionBinding
+{
+  private readonly IAppIdleManager _idleManager;
+  private const string SELECTION_EVENT = "setSelection";
+  private readonly Tekla.Structures.Model.Events _events;
+  private readonly object _selectionEventHandlerLock = new object();
+  private readonly Model _model;
+  private readonly Tekla.Structures.Model.UI.ModelObjectSelector _selector;
+
+  public string Name => "selectionBinding";
+  public IBrowserBridge Parent { get; }
+
+  public TeklaSelectionBinding(IAppIdleManager idleManager, IBrowserBridge parent)
+  {
+    _idleManager = idleManager;
+    Parent = parent;
+    _events = new Tekla.Structures.Model.Events();
+    _model = new Model();
```

Comment:
we have this model initialized in the `SpeckleTeklaPanelHost`, if we will have only one model we can inject it as singleton over there and pass it here via constructor arguments.